security/azure/sql_server_va_scans/sql_server_va_scans.pt

```diff
@@ -0,0 +1,202 @@
+name "Azure Ensure SQL Server VA Periodic Scans Enabled"
+rs_pt_ver 20180301
+type "policy"
+short_description "Report if any SQL server instances do not have Vulnerability Assessment (VA) periodic scans enabled. \n See the [README](https://github.com/flexera/policy_templates/tree/master/security/azure/sql_server_va_scans) and [docs.rightscale.com/policies](https://docs.rightscale.com/policies/) to learn more."
```

Comment:
Rightscale docs change to flexera automation docs 

"[docs.flexera.com/flexera/EN/Automation](https://docs.flexera.com/flexera/EN/Automation/AutomationGS.htm) to learn more."